client/packages.go

```diff
@@ -72,3 +72,15 @@ func (client *Client) Packages() (map[string]Package, error) {
 
 	return packages, nil
 }
+
+// Package returns details of a single Package
+func (client *Client) Package(pkgID string) (Package, error) {
```

Comment:
Do you have any information at hand that elaborates on when to use by-value and by-reference semantics?